src/index.html

```diff
@@ -7,8 +7,30 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img class="logo" src="images/logo.png"
+        alt="Moyo" >
+      </a></img>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__item"> <a class="nav_link is-active" href="#">Apple</a></li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a class="nav__link is-active" href="#">Apple</a>
          </li>
```
Fix formatting to make your code easier to read. Also, there should be two underlines in 'nav__link'.
Fix all cases